source/scripting/Vehicle.cs

```diff
@@ -1429,6 +1429,12 @@ public int PassengerCount
 				return Function.Call<int>(Hash.GET_VEHICLE_NUMBER_OF_PASSENGERS, Handle);
 			}
 		}
+        public int GetVehicleWeapon(Ped Ped)
```

Comment:
Could this return a specific type other than `int`? Is it a `WeaponHash` perhaps, or does `GET_CURRENT_PED_VEHICLE_WEAPON` give back a handle to a weapon instance?